src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -242,20 +386,20 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     const std::vector<std::string>&      metadataFilter
 )
 {
-  std::vector<SenderThreadParams*>*  paramsV;
+  std::vector<SenderThreadParams*>*  paramsV = NULL;
 
   paramsV = new std::vector<SenderThreadParams*>;
 
-  for (unsigned ix = 0; ix < cv.size(); ix++)
-  {
+  //for (unsigned ix = 0; ix < cv.size(); ix++)
+  //{
```

Comment:
Note this leftover

Reply:
Marked with "FIXME PR" in [13748bd](https://github.com/telefonicaid/fiware-orion/pull/4229/commits/13748bdedbe6a2079deb687709417d52e2de05ec).

It will be addressed (along all the remainer FIXME PR) at some moment